torch/csrc/distributed/rpc/python_functions.cpp

```diff
@@ -104,14 +104,13 @@ std::shared_ptr<RRef> pyRemoteBuiltin(
 std::shared_ptr<FutureMessage> pyRpcPythonUdf(
     RpcAgent& agent,
     const WorkerId& dst,
-    const std::string& pickledPythonUDF) {
+    const std::string& pickledPythonUDF,
+    std::vector<torch::Tensor>& tensors) {
   std::vector<char> data(pickledPythonUDF.begin(), pickledPythonUDF.end());
-  std::vector<torch::Tensor> tensor_table;
 
   return agent.send(
       dst,
-      Message(
-          std::move(data), std::move(tensor_table), MessageType::PYTHON_CALL));
+      Message(std::move(data), std::move(tensors), MessageType::PYTHON_CALL));
```

Comment:
This does not need to be in this PR. 

`ScriptCall` and `ScriptRet` has their dedicated classes. Shall we do the same for `PythonCall` and `PythonRet`? It might help distributed autograd too as `PythonCall` and `PythonRet` can then take additional fields to wrap autograd information.

Reply:
will add this in next python autograd PR